pkg/worker/handler/operator/handler.go

```diff
@@ -71,16 +64,21 @@ func New(c Config) *Handler {
 		ser = cache.New[int, service.Service]()
 	}
 
+	var clo *cloudformation.CloudFormation
 	var con *container.Container
 	var ref *reference.Reference
 	var rel *release.Release
+	var reg *registry.Registry
 	{
-		con = container.New(container.Config{Aws: c.Aws, Art: art, Env: c.Env, Log: c.Log, Ser: ser})
+		clo = cloudformation.New(cloudformation.Config{Art: art, Aws: c.Aws, Env: c.Env, Log: c.Log, Met: c.Met, Ser: ser})
+		con = container.New(container.Config{Art: art, Aws: c.Aws, Env: c.Env, Log: c.Log, Ser: ser})
 		ref = reference.New(reference.Config{Art: art, Env: c.Env, Log: c.Log, Ser: ser})
 		rel = release.New(release.Config{Art: art, Env: c.Env, Log: c.Log, Ser: ser})
+		reg = registry.New(registry.Config{Art: art, Aws: c.Aws, Env: c.Env, Log: c.Log, Ser: ser})
 	}
```

Comment:
This worker handler for the operator will now become a mere control flow structure and I still feel like our `workit` library should somehow provide this out of the box. Don't have a great idea for this at the moment.